docker-compose.yml

```diff
@@ -243,6 +243,20 @@ services:
               '--device-end-entity-cert', '/certs/first_my_cert.pem',
               '--device-private-key', '/certs/first_private_key.pem',
               '--serial-number', 'first-serial-number']
+  agent6:
+    <<: *agent
+    volumes:
+      - client-certs:/certs
+      - dhcp-leases-folder:/var/lib/dhclient/
```

Comment:
in this agent6 we only test network manager, right ?
then why we map `dhcp-leases-folder` ? I think should remove it, no ?

Reply:
Yea will remove it